src/script/components/MessagesList/Message/index.tsx

```diff
@@ -73,7 +74,7 @@ export interface MessageParams extends MessageActions {
     deleteMessageEveryone: (conversation: Conversation, message: BaseMessage) => void;
   };
   messageRepository: MessageRepository;
-  onVisible?: () => void;
+  onVisible?: (() => void) | null;
```

Comment:
Do we really need both `null` and `undefined`? Can't we stick to one falsy value? 😄